test/Interop/Cxx/class/Inputs/module.modulemap

```diff
@@ -2,6 +2,10 @@ module AccessSpecifiers {
   header "access-specifiers.h"
 }
 
+module CxxConstructors {
+  header "cxx-constructors.h"
+}
```

Comment:
The `cxx-` prefix is redundant. This is in a directory that is already called `Cxx`.

Reply:
Good point -- removed the prefix (and also in all the test names).